src/pages/main/components/product-grid/product-grid.tsx

```diff
@@ -61,7 +61,46 @@ const ProductGrid = (props: IProps) => {
           </Box>
         </>
       ) : (
-        <>Loading data ...</>
+        <>
+          <Box>
```

Comment:
why we have the same code twice?

Reply:
Because I don't have Docker installed still, so I made that in case we don't have data from BE (because I don't have Docker), it renders the static content so I can work in that section

Reply:
same reply as the previous one